vendor/experimental-render-mode-rehydrate.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  if (typeof FastBoot === 'undefined') {
+    var current = document.getElementById('fastboot-body-start');
+
+    if (current && current.nextSibling.nodeValue === '%+b:0%') {
```

Comment:
We _really really_ need a way to avoid encoding this, can you make an issue in glimmer-vm to expose a `isRehydrationComment` function that both Ember and ember-cli-fastboot can use? I'm concerned with the serialize markers changing over time (they have already changed once!?!) and then ember-cli-fastboot will have issues with cross-ember compat...

Reply:
I totally agree! I'll make the issue.